vaadin-split-layout.html

```diff
@@ -90,10 +79,9 @@
         @apply(--layout-vertical);
       }
 
-      :host > ::content > *:not(#splitter) {
+      :host ::content > * {
```

Comment:
This is what you mentioned in https://github.com/vaadin/vaadin-split-layout/pull/21, right?